src/server/implementation/utils.ts

```diff
@@ -9,7 +13,17 @@ export function stringToNumber(value: string | undefined) {
 }
 
 export async function sha256(input: string) {
-  return encodeHex(await rawSha256(new TextEncoder().encode(input)));
+  return encodeHexLowerCase(rawSha256(new TextEncoder().encode(input)));
+}
+
+export function generateRandomString(length: number, alphabet: string) {
+  const random: RandomReader = {
```

Comment:
`@oslojs/crypto` now requires the user to explicitly give a source of randomness, I’m creating it using the Web Crypto API as suggested in their documentation: https://crypto.oslojs.dev/examples/random-values